starter-code/src/App.js

```diff
@@ -1,18 +1,46 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import "./App.css";
+import "bulma/css/bulma.css";
+import foods from "./foods.json";
+import FoodBox from "./components/FoodBox";
+import FoodForm from "./components/FoodForm";
+
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    console.log(foods);
+    console.log(foods.slice(0, 5));
+    this.state = { listFood: foods.slice(0, foods.length) };
+  }
+
+  addNewFood (food) {
+    this.state.listFood.push(food)
```

Comment:
As a best practice, try to never modify the state object directly, by making copies, like so:
```js
var selectedFoodCopy = this.state.selectedFood.slice()
```